modules/canva/frontend/app/components/guac-session.js

```diff
@@ -148,6 +148,12 @@ export default Ember.Component.extend({
     let keyboard = new Guacamole.Keyboard(document);
     let display = this.guacamole.getDisplay();
 
+    window.onresize = function() {
+      let width = window.innerWidth;
+      let height = window.innerHeight;
+      this.guacamole.sendSize(width, height);
```

Comment:
Ok to trigger this function on windows resize but the size of the component should be sent to Guacamole and not the window's size.